.devcontainer/images/Dockerfile

```
@@ -0,0 +1,291 @@
+
+# =============================================================================
+# User Setup (directories + Oh My Zsh + shell config) - consolidated
+# =============================================================================
+USER vscode
+WORKDIR /home/vscode
+
+# Copy p10k configuration (referenced at runtime by .zshrc)
+COPY --chown=vscode:vscode .p10k.zsh /home/vscode/.p10k.zsh
+
+# Create cache/config directories + install Oh My Zsh + configure shells
+RUN mkdir -p .cache .config .local/bin .local/share .zsh_history_dir \
```

Hadolint (2.14.0)

[warning] In POSIX sh, brace expansion is undefined.

(SC3009)